src/fast_layers.jl

```diff
@@ -40,11 +40,25 @@ struct FastDense{F,F2} <: FastLayer
   σ::F
   initial_params::F2
   bias::Bool
+  precache::Bool

```

Comment:
there's no need to store this. If `precache=false`, then just store `nothing` and check for that in the function.